docs/root/start/building.rst

```diff
@@ -26,12 +26,30 @@ Building Envoy has the following requirements:
   :ref:`this FAQ entry <faq_build_speed>` for more information on build performance.
 * These :repo:`Bazel native <bazel/repository_locations.bzl>` dependencies.
 
-Please see the linked :repo:`CI <ci/README.md>` and :repo:`Bazel <bazel/README.md>` documentation
-for more information on performing manual builds.
 Please note that for Clang/LLVM 8 and lower, Envoy may need to be built with `--define tcmalloc=gperftools`
 as the new tcmalloc code is not guaranteed to compile with lower versions of Clang.
 
 
+Windows Requirements
+--------------------
+
+Envoy now suports Windows as a target platform. The requirements below only apply if you want to build the Windows
+native executable. If you want to build the Linux version of Envoy on Windows either with WSL or Linux containers
+please see the Linux requirements.
+
+Building Envoy has the following requirements:
```

Comment:
What are your thoughts on calling out msys here as well?  IMO it's a non-trivial requirement and is pretty key to getting the build to work...

Reply:
I was keeping it slim because unix was also keeping it slim but I agree with you. Added in f99c48964cd313ab7481a126bb11c0b0a453f7e3